storybook/stories/index.js

```diff
@@ -101,8 +97,12 @@ storiesOf('Components/Different', module)
   })
   .addDecorator((getStory) => <CenterView>{getStory()}</CenterView>)
   .add('Separator', () => <Separator />)
-  .add('SectionHeader title 1', () => <SectionHeader title='First title' />)
-  .add('SectionHeader title 2', () => <SectionHeader title='SECOND title' />)
+  .add('SectionHeaders', () => (
+    <React.Fragment>
+      <SectionHeader title='Short title' />
+      <SectionHeader title='Super long titletitletitletitletitletitle' />
```

Comment:
Sting length is not enough. Add this please: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."

Also it should be compressed into something like this (with ... at the end): "Lorem ipsum dolor sit amet ..."
The following props of Text must be applied:
```
numberOfLines=1
ellipsizeMode='tail'
```